### PR TITLE
Add message so that people know the purpose of this repo.

### PR DESCRIPTION
**Pull request by ghtstr**
- [x] Add message
- [x] Test spelling of message

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # trinug
 GitHub Essentials Test Repo
+
+super cool!
```
Comment (twiest) [inline README.md]: Consider making this uppercase.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # trinug
 GitHub Essentials Test Repo
+
```
Comment (twiest) [inline README.md]: I think it would be better to have 2 lines here.

Comment (ghtstr) [reply README.md]: Fixed

**Comment by twiest** [comment]
@ghtstr this PR needs a rebase.

**Pull request merged**